encoding/legacy-mb-japanese/iso-2022-jp/iso2022jp-decoder.js

```diff
@@ -193,8 +193,8 @@ function iso2022jpDecoder(stream) {
                         continue;
                     }
                 }
-                bytes.unshift(lead);
                 bytes.unshift(byte);
+                bytes.unshift(lead);
```

Comment:
This _might_ be more readable as:

`bytes.unshift(lead, byte);`

At least then the code order matches the spec order. On the other hand, I had to look up the behavior of `unshift()` with multiple arguments.